src/main/java/uk/gov/cslearning/record/service/scheduler/LearningJob.java

```diff
@@ -171,7 +176,16 @@ public void sendReminderNotificationForIncompleteCourses() {
         courseNotificationJobHistoryRepository.save(courseNotificationJobHistory);
     }
 
+    private void processCivilServantForCompletedCourse(CivilServant civilServant, CourseRecord courseRecord, LocalDateTime since) {
+        if (StringUtils.isNotBlank(civilServant.getLineManagerUid())) {
+            checkAndNotifyLineManager(civilServant, courseRecord, since);
+        } else {
+            LOGGER.debug("User {} has no line manager assigned. Notification skipped.", civilServant.getFullName());
```

Comment:
Should it not be warning rather then debug? 
In prod, it won't log anything to enable us to inform that why the line manager is not receiving the notification for a staff member.